registry/storage/driver/gcs/gcs.go

```diff
@@ -48,8 +48,8 @@ import (
 const driverName = "gcs"
 const dummyProjectID = "<unknown>"
 
-// driverParameters is a struct that encapsulates all of the driver parameters after all values have been set
-type driverParameters struct {
+// DriverParameters is a struct that encapsulates all of the driver parameters after all values have been set. Create a new one of these with NewDriverParameters
```

Comment:
Could you reformat these comments lines to be 99 characters or less? :neckbeard: